docs/user-guide/federation/daemonsets.md

```diff
@@ -0,0 +1,81 @@
+---
+---
+
+This guide explains how to use DaemonSets in Federation control plane.
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+This guide assumes that you have a running Kubernetes Cluster
+Federation installation. If not, then head over to the
+[federation admin guide](/docs/admin/federation/) to learn how to
+bring up a cluster federation (or have your cluster administrator do
+this for you). Other tutorials, for example
+[this one](https://github.com/kelseyhightower/kubernetes-cluster-federation)
+by Kelsey Hightower, are also available to help you.
+
+You are also expected to have a basic
+[working knowledge of Kubernetes](/docs/getting-started-guides/) in
+general and DaemonSets in particular.
+
+## Overview
+
+DaemonSets in federation control plane (referred to as "Federated Daemonsets" in
```

Comment:
Just use "Federated DaemonSets".

"Federated DaemonSets are very similar to..."